test/test_integrate.py

```diff
@@ -212,16 +213,24 @@ def _solvers():
 
 def _drift(t, y, args):
     drift_mlp, _, _ = args
-    return 0.5 * drift_mlp(y)
+    with jax.numpy_dtype_promotion("standard"):
+        return 0.5 * drift_mlp(y)
 
 
 def _diffusion(t, y, args):
     _, diffusion_mlp, noise_dim = args
-    return 0.25 * diffusion_mlp(y).reshape(3, noise_dim)
+    with jax.numpy_dtype_promotion("standard"):
+        return 0.25 * diffusion_mlp(y).reshape(3, noise_dim)
 
 
 @pytest.mark.parametrize("solver_ctr,commutative,theoretical_order", _solvers())
-def test_sde_strong_order(solver_ctr, commutative, theoretical_order):
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        jnp.float64,
+    ],
```

Comment:
Remove the comma at the end of both these lines and it should autoformat better.